modules/031-local-path-provisioner/templates/deployment.yaml

```diff
@@ -7,14 +7,23 @@ kind: VerticalPodAutoscaler
 metadata:
   name: {{ $context.Chart.Name }}-{{ $localPathProvisioner.name }}
   namespace: d8-{{ $context.Chart.Name }}
-  {{- include "helm_lib_module_labels" (list $context (dict "app" "local-path-provisioner" "workload-resource-policy.deckhouse.io" "every-node")) | nindent 2 }}
+  {{- include "helm_lib_module_labels" (list $context (dict "app" "local-path-provisioner" "workload-resource-policy.deckhouse.io" "master")) | nindent 2 }}
```

Comment:
I couldn't find any doc or comment explanation for label `workload-resource-policy.deckhouse.io`. What is its purpose? We should have a doc note for it, I believe.

Reply:
This label do not need anymore, but it will be usable to select VPA's in the cluster. `every-node` for components which deployed on any node in cluster (line cni-cilium), `master` - for components on master nodes.

Reply:
Somewhere to docs/pages/internal/DEVELOPMENT

Reply:
done